docs/design/task-DAG-scheduling.md

```diff
@@ -0,0 +1,92 @@
+# Task-level DAG scheduling policy
+
+@[hwdef](https://github.com/hwdef); July 19th, 2021
+
+## Introduction
+
+This feature provides the ability to customize the order in which tasks are launched.
+
+## Scenarios
+
+* mpi job. the master needs to wait for the worker to start before starting, If the master is already started, but the worker is not yet started, the master will restart, which will add unnecessary waste of resources ,in this case, mpiworker needs to be in the running state, and then create the pod for mpimaster..
+
+* Suppose there are two tasks and task 2 needs to use the calculation results of task 1,in this case, task1 needs to be in the complete state, and then create the pod for task2.
+
+* some ETL applications
+
+## Requirement
+
+Each task can specify which one or more tasks it depends on, and it can specify the time point or interface for triggering.
+
+Examples of time points:
+
+* running
```

Comment:
IC, Are there any other time points? Suggest to list all possbible time points in order to take them into consideration when doing feature design.

Reply:
I can't think of another time pointes at the moment.